src/browser/modules/Stream/SysInfoFrame/legacyHelpers.tsx

```diff
@@ -236,41 +236,3 @@ export const responseHandler = (setState: any) =>
       success: true
     })
   }
-
-export const clusterResponseHandler = (setState: any) =>
```

Comment:
None of this cluster code was used? We should probably check that things are working properly with a server cluster... :-)

Reply:
In the PR description I've linked the PR that motivated this change :)